utils/convertParams.ts

```diff
@@ -1,7 +1,7 @@
 export const objToURLSearchParams = (params: object) => {
   const flattenedArray: string[][] = [];
   for (const [key, value] of Object.entries(params)) {
-    if (!value) continue;
+    if (value === undefined) continue;
```

Comment:
```suggestion
    if (value === undefined || value === null) continue;
```
생각해보니 undefined 말고 null이 들어가 있을 수도 있을 듯,,, 사실 0만 패스해줘야 하는 거니까 이렇게 `null`도 추가해주시면 감사하겠습니다ㅏ

Reply:
이거 null도 유효한 value라서 체크 안했던거 아닌가?

Reply:
아 쿼리에 null 넣는 경우가 있었던가...?! 그러면 일단 이대로 유지해도 될듯!